src/payment/models/bank-tx/sepa-parser.service.ts

```diff
@@ -81,6 +81,7 @@ export class SepaParser {
           ...this.getTotalCharge(entry?.NtryDtls?.TxDtls?.Chrgs?.Rcrd),
           ...this.getRelatedPartyInfo(entry),
           ...this.getRelatedAgentInfo(entry),
+          accountIban: Config.bank.maerkiBaumann.ibanEur,
```

Comment:
Use the IBAN from the batch (there are also multiple accounts), e.g. as method parameter.